src/main/java/hudson/plugins/sonar/client/ProjectInformation.java

```diff
@@ -54,11 +53,10 @@ public String getCeStatus() {
     return ceStatus;
   }
 
-  public void setCeStatus(@Nullable String ceStatus) {
+  public void setCeStatus(String ceStatus) {
     this.ceStatus = (ceStatus != null) ? ceStatus.toLowerCase(Locale.US) : null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this condition so that it does not always evaluate to "true" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2583)